test/testinit.js

```diff
@@ -139,7 +139,7 @@ TestManager = {
 TestManager.init( {
 	"jquery": {
 		urlTag: "jquery",
-		choices: "dev,min,git,3.3.1,3.2.1,3.1.1,3.0.0"
+		choices: "dev,min,git,3.x-git,3.3.1,3.2.1,3.1.1,3.0.0"
```

Comment:
I left the `git` choice here but it's no longer run against `master` on TestSwarm. I assume it's fine for now, until we start making Migrate compatible with what Core 4.0 brings?

Reply:
I think this is fine for now.